src/pages/Home.jsx

```diff
@@ -1,24 +1,36 @@
-import React, { useEffect } from 'react';
+import React, {useState} from 'react';
 import PageTemplate from "../components/PageTemplate/PageTemplate";
 import CustomHeading from "../components/CustomHeading";
-import StockSearchBar from '../components/StockTable/StockSearchBar';
-import StockTableCard from '../components/StockTable/StockTableCard';
-import { useStockList } from '../contexts/useStockList';
+import {VStack} from "@chakra-ui/react";
+import StockSearchBar from "../components/StockTable/StockSearchBar";
+import StockInfoTable from "../components/StockTable/StockInfoTable";
+import {useStockSymbol} from "../contexts/useStockInfo";
 
 const Home = () => {
-    const { setStockList } = useStockList();
+    const {basicStockInfo, filterStocks} = useStockSymbol()
+    const [filterKeyword, setFilterKeyword] = useState()
 
-    useEffect(() => {
-        setStockList();
-    }, []);
+    const handleChange = (event) => {
+        event.preventDefault();
+        setFilterKeyword(event?.target?.value)
+        filterStocks(event?.target?.value)
+    }
+
+    const handleCancel = event => {
+        event.preventDefault()
+        setFilterKeyword('')
+        filterStocks('')
+    }
 
     return (
         <PageTemplate>
-            <CustomHeading>Stock Market Overview</CustomHeading>
-            <StockSearchBar />
-            <StockTableCard />
+            <VStack spacing={16} m={4}>
```

Comment:
change VStack to CustomBox component that I made? and bring CustomHeading out

Reply:
We dnt need a box here 
We need a Vstack